### ISSUE TITLE
feature request: add windef to OpenPIVGui

### ISSUE THREAD

**Issue by alexlib** [OpenPIV/openpiv_tk_gui#7]
Hi, @eguvep 

is it possible to add windef to the TK GUI ? I've now encountered a set of images that are not as good for regular single pass PIV, yet I get a reasonable result after windef. 

https://github.com/OpenPIV/openpiv-python/blob/master/openpiv/examples/tutorials/example_windef_run.py
https://github.com/OpenPIV/openpiv-python/blob/master/openpiv/examples/notebooks/window_deformation_comparison.ipynb

BTW, we already have a 3D version of PIV, so the next feature request is on the way :) Seriously only if someone has time to look at it. 

Thanks in advance
Alex

**Comment by eguvep** [comment]
Hi @alexlib,

I just implemented windef into the openpiv-tk-gui and I am impressed: Very fast and precise!

For now, I kept the parameters in the GUI at a minimum and hard coded most useful values for most cases or left the default values as they are in windef.py. Tell me, if you think I should add one or more parameters specific to windef. It is pretty easy and quickly done for most cases.

Regards,

Peter
